packages/at_client/lib/src/preference/at_client_preference.dart

```diff
@@ -66,6 +66,9 @@ class AtClientPreference {
 
   Duration monitorHeartbeatInterval = Duration(seconds: 10);
 
+  /// Time interval for the scheduled task that removes expired keys from local keyStore
+  int expiryCheckTimeIntervalMins = 10;
```

Comment:
Can you make this a Duration please, allows for more flexibility

Reply:
chnaged as part of https://github.com/atsign-foundation/at_client_sdk/pull/1187/commits/bf650bef4d6bd58f36e67e0d55ca5bf26fe95e5b